GLTF/GLTFAsset.h

```diff
@@ -56,10 +57,20 @@ namespace GLTF
 
     typedef std::map<std::string, std::shared_ptr<GLTFOutputStream> > NameToOutputStream;
 
+	struct UniqueIdMesher {
```

Comment:
I'm pretty sure OpenCOLLADA already has a defined hash function for UniqueId: https://github.com/KhronosGroup/OpenCOLLADA/blob/master/COLLADAFramework/src/COLLADAFWUniqueId.cpp#L160

If you think this performs better/gives a better distribution than what they have already I would encourage you to open a pull request into that repo, but doing it here is a bit out of the scope of this project.